### PR TITLE
README: add badges, links and some content

### PR DESCRIPTION
**Pull request by amery**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,68 @@
...
+[darvaza-sidecar]: https://pkg.go.dev/darvaza.org/sidecar
+[darvaza-simplelru]: https://pkg.go.dev/darvaza.org/cache/x/simplelru
+[darvaza-x-config]: https://pkg.go.dev/darvaza.org/x/config
+[darvaza-x-tls]: https://pkg.go.dev/darvaza.org/x/tls
+[darvaza-x-web]: https://pkg.go.dev/darvaza.org/x/web
+
+## Dependencies
+
+The _Darvaza Extra_ modules are built on top of a handful
+low(ish) level packages additionally to the Go Standard Library.
```
Comment (karasz) [inline README.md:28]: in addition to the Go Standard....

File: README.md [hunk 2]
```diff
@@ -1,25 +1,68 @@
...
+
+* Our _core_ package,
+  [`darvaza.org/core`][darvaza-core].
```
Comment (karasz) [inline README.md:31]: please add a sentence of description to `core`

File: README.md [hunk 3]
```diff
@@ -1,25 +1,68 @@
...
+  [`darvaza.org/cache/x/simplelru`][darvaza-simplelru].
 
 ## Packages
 
-### darvaza.org/x/config
+### Config
 
-[darvaza.org/x/config](https://daravaza.org/x/config) provides helpers
+[`darvaza.org/x/config`][darvaza-x-config] provides helpers
 for reading config files.
```
Comment (karasz) [inline README.md:43]: I would say `dealing`, not `reading` because afair the `Config` also writes, dumps configs

**Review by karasz — CHANGES_REQUESTED** [review]

**Comment by amery** [comment]
amended @karasz

**Pull request merged**